examples/lock-app/efr32/README.md

```diff
@@ -19,13 +19,21 @@ An example showing the use of CHIP on the Silicon Labs EFR32 MG12.
 
 ## Introduction
 
-The EFR32 lock example provides a baseline demonstration of a door lock device,
```

Comment:
A bunch of the same typos here as in lighting-app; please fix them here as well

Reply:
It should be good now. I spell-checked both readmes